RELEASING.md

```diff
@@ -120,3 +126,19 @@ gem.
    [`google-cloud/google-cloud.gemspec`](https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud/google-cloud.gemspec).
 1. Add the gem to
    [`gcloud/Gemfile`](https://github.com/googleapis/google-cloud-ruby/blob/master/gcloud/Gemfile).
+
+## Checking the status of kokoro builds
+
+1. On the [commits](https://github.com/googleapis/google-cloud-ruby/commits/master) page, find the commit you are checking for.
+
+1. To the right of your commit, there should be either a red x, or a green checkmark. Click it.
+
+1. A modal will appear with the list of kokoro checks. To learn more about why a check failed or what it was testing for click "details" next to the check.
+
+1. The "details" link will take you to a page on https://source.cloud.google.com/. On the "TARGETS" tab, there will be a link to the logs. It will look like "cloud-devrel/client-libraries/google-cloud-ruby/#{job_type}-#{operating_system}". Click the link.
+
+1. The next page will have a list of all the tasks carried out under the selected operating system. The top half will be a list of tasks and the links to their respective logs, the bottom half will be a list of tasks and their status (SUCCEEDED/FAILED). To learn more about why a task failed, clink the corresponding link on the top half of the screen.
+
+1. You will be greeted with a screen similar to the one in step 4. The link to your task should look like "cloud-devrel/client-libraries/google-cloud-ruby/#{job_type}/#{operating_system}/#{task_name}". Click it.
+
+7. The "TARGET LOG" tab should be pre-selected, and will contain the complete logs.
```

Comment:
7. -> 1.

Reply:
Now what? Maybe: "Find the error. If you suspect it is a transient error, restart the build job (how??). If it is a deterministic error requiring a code update, you will need to correct the problem with a new pull request, and possibly issue a follow-up release (more detail? more instructions??)"